test/fixtures/order_of_end_tags_5873.markdown

```diff
@@ -0,0 +1,6 @@
+# Title
```

Comment:
I think there was a proposal to use the extension `.md` instead of `.markdown`. cc @nodejs/documentation